app/src/main/java/com/infomaniak/mail/data/api/FlatteningSubBodiesSerializer.kt

```diff
@@ -22,7 +22,7 @@ import io.realm.kotlin.serializers.RealmListKSerializer
 import io.realm.kotlin.types.RealmList
 import kotlinx.serialization.json.*
 
-// Documentation: https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json-transforming-serializer/
+// Documentation on KotlinLang.org : https://chk.me/kYZiqN0
```

Comment:
Not sure about that solution as well

Reply:
What do you propose instead ?

Reply:
Yeah the comment wasn't very clear now that I read it again. I think we could keep that one on a single line as it doesn't obstruct reading